lib/sda/host_msg.go

```diff
@@ -61,10 +67,13 @@ type TokenID uuid.UUID
 type Token struct {
 	EntityListID EntityListID
 	TreeID       TreeID
-	ProtoID      ProtocolID
-	RoundID      RoundID
-	TreeNodeID   TreeNodeID
-	cacheId      TokenID
+	// TO BE REMOVED
```

Comment:
ServiceID and RoundID will be kept in the Token for sure (and the token is given to a TreeNodeInstance, as we are doing now with Node). I am not sure if we still need the ProtocolID since we have already a TreeNodeID , TreeID + EntityListID. I still have to see if that is sufficient to uniquely identify a running PI.